client/src/App.js

```diff
@@ -303,8 +331,50 @@ export default class NormalDisplay extends React.Component {
                 </pre>
     }
 
+    async checkJournalReady(journalUrl) {
+        if (this.isAborted) return;
+        let resp;
+        try {
+            var data = new FormData();
+            data.append('journalUrl',journalUrl);
+            resp = await fetch("/api/check-jrnl-ready", {
```

Comment:
Do we really need to abreviate `journal` with `jrnl` here in the API endpoint? I mean `check-journal-ready` is not toooo long (btw it reminds more of an rpc-type url because of the verb :) "check").

Reply:
thanks 👍